### ISSUE TITLE
WoRMS (via Arctos) refresh isn't refreshing

### ISSUE THREAD

**Issue by sharpphyl** [ArctosDB/arctos#5686]
I requested a refresh of the genus Turcica.

![check status request](https://user-images.githubusercontent.com/15368365/220428784-17250e56-eddf-4469-adb1-c55f5d237d42.JPG)

Clicking on Check Status appears to show that the refresh is complete.

![Check Status result](https://user-images.githubusercontent.com/15368365/220428920-c66d96f7-7eaf-48f4-8abf-663b9b8cd2d5.JPG)

A check of one of the Turcica species shows that it has not been refreshed.

![Turcica angulifera before refresh](https://user-images.githubusercontent.com/15368365/220429047-f4b1e0ae-529e-450c-b2cd-91df7f5e81f7.JPG)

I'm having to manually refresh each record individually to update the family from Eucyclidae to Turcicidae

![Turcica angulifera after manual refresh](https://user-images.githubusercontent.com/15368365/220429108-c5fcc9e2-8693-490a-aba4-ac3d181e4bec.JPG)

I requested a refresh of the superfamily [Seguenzioidea](https://marinespecies.org/aphia.php?p=taxdetails&id=224565) containing Turcica yesterday and it appears nothing was refreshed.

**Comment by dustymc** [comment]
I can't see how this might work in the code, looks happy to me and it's all using the same function, but maybe if you've got something that needs refreshed I can try it?

**Comment by sharpphyl** [comment]
I think you just reset my account.  I was able to log in in less than a minute for the first time in a month.  Thank you!  

I set Eucyclidae to refresh.  There's still nothing in the "check status" screen (which usually shows how many records have been refreshed and how many remain to be refreshed) but I'll check later and let you know if the WoRMS (via Arctos) data has been refreshed.  If you want to experiment with a different group, I was about to refresh the superfamily [Patelloidea](https://www.marinespecies.org/aphia.php?p=taxdetails&id=156481).
